app/controllers/sales_logs_controller.rb

```diff
@@ -32,12 +34,26 @@ def show
   end
 
   def edit
-    @log = current_user.sales_logs.visible.find_by(id: params[:id])
-    if @log
-      render "logs/edit", locals: { current_user: }
-    else
-      render_not_found
-    end
+    @log = current_user.sales_logs.visible.find(params[:id])
+    render "logs/edit", locals: { current_user: }
+  end
+
+  def destroy
```

Comment:
if we're actually discarding, shouldn't we call this method and the one in log policy discard too?